static/examples/euid-cstg-prebid-example/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>EUID Prebid.js Client-Side Integration Example</title>
+    <link rel="stylesheet" type="text/css" href="./app.css" />
+    <link rel="shortcut icon" href="/img/favicon.ico" />
+    <script async src="./prebid.js"></script>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+      console.log('Initializing example.')
+      window.euid_example_settings = {
+        EUID_BASE_URL: 'https://operator-integ.uidapi.com',
```

Comment:
This is the UID2 integ operator - I think for thoroughness here we should point to EUID infra. The EUID operator is at https://integ.euid.eu